app/User.php

```diff
@@ -4,15 +4,18 @@
 
 use Carbon\Carbon;
 use Illuminate\Foundation\Auth\User as Authenticatable;
+use League\OAuth2\Client\Token\AccessToken;
+use App\Http\Controllers\VatsimOAuthController;
 use Illuminate\Notifications\Notifiable;
 use Laratrust\Traits\LaratrustUserTrait;
 
 class User extends Authenticatable {
     use Notifiable;
     use LaratrustUserTrait;
     protected $table = 'roster';
-    protected $fillable = ['id', 'fname', 'lname', 'email', 'rating_id', 'canTrain', 'visitor', 'status', 'loa', 'del', 'gnd', 'twr', 'app', 'ctr', 'train_pwr', 'monitor_pwr', 'opt', 'initials', 'added_to_facility', 'max','max_minor_del','max_minor_gnd','max_minor_twr','max_minor_app','twr_solo_fields','twr_solo_expires'];
+    protected $fillable = ['id', 'fname', 'lname', 'email', 'rating_id', 'canTrain', 'visitor', 'status', 'loa', 'del', 'gnd', 'twr', 'app', 'ctr', 'train_pwr', 'monitor_pwr', 'opt', 'initials', 'added_to_facility', 'max','max_minor_del','max_minor_gnd','max_minor_twr','max_minor_app','twr_solo_fields','twr_solo_expires','access_token','refresh_token','token_expires'];
```

Comment:
Splitting my next comment into 2 comments...

As a matter of fact, I'm pretty sure the entire fillable array can be removed to clean things up a little bit. I'm pretty sure Laravel will assume all columns on the table to be the fillable if we don't provide it.

Reply:
Yes that should work. I'm curious what the default is if we leave it out (the parent eloquent Model) should have it implemented some way.